vllm/model_executor/layers/fused_moe/rocm_aiter_fused_moe.py

```diff
@@ -18,6 +18,50 @@ def is_rocm_aiter_block_scaled_moe_enabled() -> bool:
         envs.VLLM_ROCM_USE_AITER_FP8_BLOCK_SCALED_MOE
 
 
+def is_rocm_aiter_channel_scaled_moe_enabled() -> bool:
+    return is_rocm_aiter_moe_enabled() and \
+        envs.VLLM_ROCM_USE_AITER_FP8_CHANNEL_SCALED_MOE
+
+
+def rocm_aiter_asm_moe_tkw1(hidden_states,
+                            w1,
+                            w2,
+                            topk_weight,
+                            topk_ids,
+                            fc1_scale=None,
+                            fc2_scale=None,
+                            fc1_smooth_scale=None,
+                            fc2_smooth_scale=None,
+                            a16=False,
+                            per_tensor_quant_scale=None,
+                            expert_mask=None,
+                            activation_str: str = "silu") -> None:
+
+    from aiter import ActivationType
+    from aiter.fused_moe_bf16_asm import asm_moe_tkw1
+
+    if activation_str == "silu":
+        activation = ActivationType.Silu
+    elif activation_str == "gelu":
+        activation = ActivationType.Gelu
+    else:
+        activation = ActivationType.Silu
```

Comment:
Can be simplified to one-liner ?

```suggestion
    activation = ActivationType.Gelu if activation_str == "gelu" else ActivationType.Silu
```

Reply:
Do we need an additional wrapper for the _tkw1 kernel, given that it’s just a kernel call plus an activation type conversion? the activation type can also used by other branches / kernel calls?

Reply:
We are wrapping the kernel call because in our future PR addressing the enablement of torch compile for aiter MoE kernels, we will be using wrappers to register the aiter ops, and so we thought to leave it here for now.